src/applications/caregivers/components/PreSubmitInfto/test-data.js

```diff
@@ -0,0 +1,80 @@
+export const formData = {
```

Comment:
This was test data used to scaffold form before we had a schema in `vets-json-schema` can probably remove this...